src/auth-utils.php

```diff
@@ -14,27 +14,25 @@
  */
 function init_global_admin_tools_auth() {
 
-	$auth_data = [
-		'site_url' => 'default',
-		'username' => 'easyengine',
-		'scope'    => 'admin-tools',
-	];
-
-	if ( ! empty( Auth::where( $auth_data ) ) ) {
-		EE::log( 'Global auth exists on admin-tools. Use `ee auth list global` to view credentials.' );
+	if ( ! empty( Auth::get_global_admin_tools_auth() ) ) {
+		EE::log( 'Global auth exists on admin-tools. Use `ee auth list global_admin_tools` to view credentials.' );
```

Comment:
This log message seems wrong. Should it not be: __Use `ee auth list global` to view credentials.__